tests/data/bundle.yaml

```diff
@@ -8,7 +8,4 @@ applications:
     scale: 1
```

Comment:
should we keep using this bundle file even though we only need to deploy the charm now and nothing else?

Reply:
sure, i can toss it out